data/pubspec.yaml

```diff
@@ -0,0 +1,29 @@
+name: data
+description: Data layer
+version: 0.0.1
+
+publish_to: 'none'
+
+environment:
+  sdk: ">=2.16.1 <3.0.0"
+
+dependencies:
+  domain:
+    path: ../domain
+
+  dio: ^4.0.6
+  equatable: ^2.0.3
+  get_it: ^7.2.0
```

Comment:
Should not be here - all dependencies should be build in presentation layer

Reply:
del getIt